src/pybind/mgr/dashboard/frontend/src/app/shared/api/cephfs.service.spec.ts

```diff
@@ -17,4 +21,55 @@ describe('CephfsService', () => {
       expect(service).toBeTruthy();
     })
   );
+
+  it(
+    'should call getCephfs',
+    inject(
+      [CephfsService, HttpClient],
+      fakeAsync((service: CephfsService, httpClient: HttpClient) => {
+        let count = 0;
```

Comment:
Each test is using a count variable - how about setting it to zero during in a `beforeEach`?

(And on any other line something like this would make sense)

Reply:
Done